mmv1/api/async.go

```diff
@@ -83,12 +83,8 @@ type OpAsync struct {
 type OpAsyncOperation struct {
 	Kind string `yaml:"kind,omitempty"`
 
-	Path string `yaml:"path,omitempty"`
-
 	BaseUrl string `yaml:"base_url,omitempty"`
 
-	WaitMs int `yaml:"wait_ms,omitempty"`
-
```

Comment:
I might need to leave it in the serializer otherwise this will break any existing PRs that use this.